### ISSUE TITLE
Some errors  that never appeared  when building for quest  in logger.cpp .

### ISSUE THREAD

**Issue by Hope10086** [korejan/ALVR#84]
#82  #81  's follow-up:

Yesterday, I tried to build the alxr -client - quest from the pulled branch:[`facial-eye-refactor `](https://github.com/korejan/ALVR/tree/facial-eye-refactor)source code, but the following error occurred in alxR-engine-sys.
```
:app:buildCMakeDebug[arm64-v8a]-2
logger.cpp\..\ALVR-OpenXR-Engine\src\alxr_engine\logger.cpp
use of undeclared identifier 'Level'; did you mean 'Log::Level'?
no member named 'str' in 'std::basic_string_view<char>'
expanded from macro 'ALOGE'
no member named 'str' in 'std::basic_string_view<char>'
expanded from macro 'ALOGV'
```
I did not encounter such errors when building` alxr-client-quest` for master and [`facial-eye-tracking` ](https://github.com/korejan/ALVR/tree/facial-eye-tracking)previously. I compared `logger.cpp `in different branches and found no obvious problems.

Regarding the use of the[XR_FB_eye_tracking_social](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html) extension in `facial-eye-tracking`, I would like to know when the macro: `XR_USE_OXR_OCULUS` in the code is declared.  We just bought the quest pro and wanted to compare the differences  of `GazePoses` between  using XR_FB_eye_tracking_social  and  [XR_EXT_eye_gaze_interaction](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html) .

**Comment by korejan** [comment]
@Hope10086 hey, sorry about that I've pushed fixes for compile errors and rebased the branch with latet, I've changed default display colour space for Quest headsets from Rec.2020 to Quest colour space, should work better with local dimming too.

This define is set in cmake fles and set by rust cargo features feed to the rust build script. It has necessary because the facebook facial/eye tracking extensions were previously unpublished extensions not in the offical OpenXR SDK headers and required either getting extra headers from the Oculus Mobile OpenXR SDK or manually hand-writing the types & signatures.

These extensions have now been made published since about a week and are in v1.0.27 OpenXR spec & SDK so once I upgrade to this version these macros can go.
